architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -0,0 +1,89 @@
+[[architecture-additional-concepts-ephemeral-storage]]
+= Ephemeral Local Storage
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+In addition to persistent storage, pods and containers may require
+ephemeral (transient) local storage for their operation.  The lifetime
+of this ephemeral storage does not extend beyond the life of the
```

Comment:
Ephemeral storage is also tied to emptyDir volumes so it’s life of pod and not Container.